cmd/govim/config/config.go

```diff
@@ -151,6 +151,14 @@ type Config struct {
 	// GOFLAGS=-modfile=go.local.mod in order to use an alternative go.mod file.
 	GoplsEnv *map[string]string `json:",omitempty"`
 
+	// Analyses is a map of booleans (0 or 1 in VimScript) used to enable/disable
+	// specific analyses in gopls.
```

Comment:
I think we need to expand this comment a bit to indicate that it's only the presence of an entry in the map that enables/disables an analysis. i.e. if I supplied an empty map for the `Analyses` config field then the default setting for each respective analysis would be used (according to the `gopls` docs).

Indeed it might even be worthwhile linking to the specific version of the `gopls` docs we are pinned to (although this could follow in a later PR)

Reply:
Ack, I'll add a link to master for this PR.